src/homepage/index.html

```diff
@@ -73,7 +73,7 @@ <h2>About</h2>
               <div class="about-table">
                 <h3 class="tb-h3">James from a technical point of view</h3>
                 <b>Complete portability</b> (100% pure Java)<br/>
-                <b>Running on Java 6 onwards</b> and on the <b>JVM</b>. Some optional components might require Java 8.<br/><br/>
+                <b>Built with Java 8</b> and running on the <b>JVM</b>.<br/><br/>
```

Comment:
Don't we require java 8 for runnig as well? (`<target.jdk>1.8</target.jdk>`)

Reply:
s/JVM/JRE 1.8/ ?